aria/builtins/workflows.py

```diff
@@ -104,6 +104,8 @@ def install_node_instance_sub_workflows(context, graph, node_instance):
     ])
```

Comment:
please move all the doto's from this file to some place else that is not in the code